### DISCUSSION TITLE
Low-resource language

### DISCUSSION THREAD

**Discussion by EklavyaFCB** [openai/whisper#472] [Q&A]
Hello,
I want to transcribe a low-resource language which is not included in the languages list in the paper. 
Would this still somehow be possible ? 
Just getting phonemes rather than whole words is also useful. 
It's a south American language similar to Spanish.

**Comment by jongwook** [comment]
While it'd be better to fine-tune on some labeled data (e.g. https://huggingface.co/blog/fine-tune-whisper), have you tried just specifying Spanish and run Whisper? When we tried [Asturian](https://en.wikipedia.org/wiki/Asturian_language), another language adjacent to Spanish, it worked better than random.
